spec/policies/event_policy_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe EventPolicy do
+
+  let(:reviewer) { create(:user, :reviewer) }
+  let(:organizer) { create(:organizer) }
+  let(:admin) { create(:admin) }
+  let(:event) { create(:event) }
+
+  subject { described_class }
+
+  #permissions ".scope" do
+  #  pending "add some examples to (or delete) #{__FILE__}"
+  #end
+
+  permissions :show?, :create?, :update?, :destroy? do
+    it 'denies reviewer users' do
+      expect(subject).not_to permit(reviewer, event)
+    end
+
+    it 'allows admin users' do
+      expect(subject).to permit(admin, event)
+    end
+
+    it 'allows organizer users' do
+      org_event = organizer.organizer_events.first
+      expect(subject).to permit(organizer, org_event)
+    end
+
```

Comment:
I'm not sure how public event pages fit into this policy but all users including anonymous ones can access the public event pages.